clients/src/test/java/org/apache/kafka/common/utils/LogCaptureAppender.java

```diff
@@ -80,6 +86,17 @@ public static LogCaptureAppender createAndRegister(final Class<?> clazz) {
         return logCaptureAppender;
     }
 
+    public void setClassLoggerToInfo(final Class<?> clazz) {
```

Comment:
Could we add new method `setClassLogger` to replace those `setClassLoggerToXXX`?
```java
    public void setClassLogger(final Class<?> clazz, Level level) {
        logLevelChanges.add(new LogLevelChange(Logger.getLogger(clazz).getLevel(), clazz));
        Logger.getLogger(clazz).setLevel(level);
    }
```

That is more simple and not verbose to me

Reply:
Agree, updated it. Thank you.